docs/sources/reference/loki-http-api.md

```diff
@@ -438,6 +438,35 @@ curl -u "User:$API_TOKEN" \
   --data-urlencode 'query=sum(rate({job="varlogs"}[10m])) by (level)' | jq
 ```
 
+### Response Format
```

Comment:
I'm not sure if this is the best place for this documentation. It feels like that the response schemas belong to their respective query type sections.

Reply:
I agree. This is in the section but maybe not the right spot. @JStickler what do you think?

Reply:
Ooooh, that's a tough one.  I think that chaudum has the right idea, that the response schemas belong with their respective query types.

If you think this is the correct section, I'd at least move your addition up above the examples, either to line 300 or line 350.